### ISSUE TITLE
Mastodon instance not sending emails because of error in the installation

### ISSUE THREAD

**Issue by byKeksYT** [mastodon/mastodon#22484]
### Steps to reproduce the problem

1. Install Mastodon via the steps provided [here](https://docs.joinmastodon.org/admin/install/)
2. When it gets to the `RAILS_ENV=production bundle exec rake mastodon:setup` command and there to the email part, create an address for notifications
3. Test your newly created email adress by sending a test email
...

### Expected behaviour

No error happening, the email for notifications to work normally

### Actual behaviour

Error with sending the test email: `E-mail could not be sent with this configuration, try again. Connection refused - connect(2) for "localhost" port 25`

### Detailed description

My email address that I created for notifications caanot be used because Mastodon cannot access the locally hosted mail server.

### Specifications

The server is using following software:

Debian 11 (Bullseye)
Mastodon v4.0.2
Node.js v16.18.1

**Comment by ineffyble** [comment]
Do you have a locally hosted mail server definitely running on port 25?

**Comment by byKeksYT** [comment]
Does the setup not provide a mail server? I thought it would install all things that are neccessary for the instance.

**Comment by ineffyble** [comment]
No, it's listed as one of the 3 prerequisites in the page you linked:

<img width="855" alt="image" src="https://user-images.githubusercontent.com/834880/208443052-94e0ded8-ac33-4cc5-8c3b-ceac66a6fda1.png">

**Comment by byKeksYT** [comment]
Oh okay, then I will install a mail server. What do I have to do in order for Mastodon to use it? In the admin interface I did not find an option for mails or a mail server.

**Comment by ineffyble** [comment]
SMTP config goes in Mastodon's `.env.production`: https://docs.joinmastodon.org/admin/config/#smtp

If you're installing an SMTP server on the same server, it's likely the defaults will work for you. Whether that server is able to deliver email anywhere is another question, since many hosts block outgoing port 25 due to spam.
